src/ceph-volume/ceph_volume/devices/lvm/strategies/filestore.py

```diff
@@ -167,22 +166,28 @@ class MixedType(MixedStrategy):
     SSDs are used, otherwise it will be used directly.
     """
 
-    def __init__(self, devices, args):
-        super(MixedType, self).__init__(devices, args)
+
+    def __init__(self, data_devs, db_devs, args):
```

Comment:
this is filestore, this probably needs journal_devs here instead of db_devs

Reply:
Yes, will fix this.